docker-compose.prod.yml

```diff
@@ -0,0 +1,28 @@
+version: '3'
+
+services:
+  db:
+    image: 'postgres:10.5-alpine'
+    container_name: 'db'
+    volumes:
+      - './tmp/db:/var/lib/postgresql/data'
+    env_file:
+      - './api/.env'
+  api:
+    depends_on:
+      - 'db'
+    container_name: 'tapp_api'
+    command: bundle exec rails s -p 3000 -b "0.0.0.0"
+    build: ./api
+    ports:
+      - '3000:3000'
+    env_file:
+      - './api/.env'
+  frontend:
+    container_name: 'tapp_fe'
```

Comment:
The frontend container does not need to run in production because the frontend is served by the rails app.